src/localize/languages/pt_br.json

```
@@ -210,8 +216,8 @@
     "invalid_configuration": "Configuração inválida",
     "invalid_configuration_no_hint": "Nenhuma dica de local disponível (tipo incorreto ou ausente?)",
     "upgrade_available": "Uma atualização de configuração de cartão automatizada está disponível, visite o editor de cartão visual",
-    "webrtc_reported_error": "O componente WebRTC relatou um erro",
-    "webrtc_waiting": "Aguardando o carregamento do cartão WebRTC ...",
+    "webrtc_card_reported_error": "O componente WebRTC relatou um erro",
+    "webrtc_card_waiting": "Aguardando o carregamento do cartão WebRTC ...",
```

```suggestion
    "webrtc_card_waiting": "Aguardando o cartão WebRTC carregar...",
```

I don't know. Any second opinions?